tilequeue/query/__init__.py

```diff
@@ -1,4 +1,9 @@
-from tilequeue.query import postgres, fixture
+from tilequeue.query.fixture import make_data_fetcher
+from tilequeue.query.pool import DBConnectionPool
+from tilequeue.query.postgres import make_db_data_fetcher
 
-make_db_data_fetcher = postgres.make_db_data_fetcher
-make_fixture_data_fetcher = fixture.make_data_fetcher
+__all__ = [
+    'DBConnectionPool',
+    'make_data_fetcher',
```

Comment:
I was using the assignment to rename `fixture.make_data_fetcher` to `make_fixture_data_fetcher`. Is that possible with the `__all__` re-export, or should we just rename the function in the `fixture.py` file?